src/Raven.Server/Web/System/DatabaseHelper.cs

```diff
@@ -23,7 +25,7 @@ private static string[] GetServerWideOnlyConfigurationKeys()
 
                 foreach (var categoryProperty in configurationProperty.PropertyType.GetProperties(BindingFlags.Instance | BindingFlags.Public))
                 {
-                    var configurationEntryAttribute = categoryProperty.GetCustomAttribute<ConfigurationEntryAttribute>();
+                    var configurationEntryAttribute = categoryProperty.GetCustomAttributes<ConfigurationEntryAttribute>().FirstOrDefault();
```

Comment:
Both of the keys should be inserted, not only the first.

Also check 'RavenConfiguration.AllConfigurationKeys'. We need to make changes there also.